### ISSUE TITLE
Sorry, but there is no formatter for 'twig'-files installed?

### ISSUE THREAD

**Issue by keithmancuso** [mblode/vscode-twig-language#11]
I think i have everything setup right but its not formatting on save or when I manually call "Format Document"?  It just says "Sorry, but there is no formatter for 'twig'-files installed."

Im sure im missing something dumb

**Comment by richjava** [comment]
I'm getting the same thing.

**Comment by mblode** [comment]
Are you still having this issue? If so, can you explain your setup so I can recreate the bug? Thank you 😄

**Comment by davorpeic** [comment]
Yeah, I think the issue is related with installing your "Twig language", which does give error message others are reported. I fix the problem installing "Twig language 2" which you made I suppose to correct this issue https://marketplace.visualstudio.com/items?itemName=mblode.twig-language-2

Maybe you could add some disclaimer message at the beginning of first version?
dp

**Comment by kuworking** [comment]
Definitively it is not clear what should be installed, whether the two plugins or only one of them

In my case this has worked after restarting vscode and disabling the "other" twig extension (from whatwedo) (one of these two things solved the problem)

That said, wonderful and very needed extension, big thanks (!!)

**Comment by Ibrahima-prog** [comment]
> Definitively it is not clear what should be installed, whether the two plugins or only one of them
> 
> In my case this has worked after restarting vscode and disabling the "other" twig extension (from whatwedo) (one of these two things solved the problem)
> 
> That said, wonderful and very needed extension, big thanks (!!)

I reloaded the window and it worked!! otherwise, VS Code was telling me that no twig formatter was installed.

Maybe the author should mention that in the documentation.

P.S: I am working on docker containers.

**Comment by mblode** [comment]
Twig Language still uses HTML mode in 0.10.0. For a dedicated Twig language mode, use Twig Language 2; both share the formatter.
